src/screens/token/components/delegate-token/components/tabs/validators-tab/validators-tab.tsx

```diff
@@ -106,8 +105,12 @@ export class ValidatorsTabComponent extends React.Component<
                         />
                     </View>
                     {this.state.validatorsFilteredList.length === 0 ? (
-                        <View style={{ flex: 1 }}>
-                            <LoadingIndicator />
+                        <View style={styles.emptySection}>
+                            <Image
```

Comment:
Here you should have all the 3 states:

1. Loading
2. Empty state
3. Validators list

We can have a quick call to explain